apps/native-component-list/src/screens/ModulesCore/ModulesCoreScreen.tsx

```diff
@@ -20,15 +20,15 @@ export const ModulesCoreScreens = [
   },
 ];
 
-if (!isRunningInExpoGo()) {
-  ModulesCoreScreens.push({
-    name: 'Benchmarks',
-    route: 'modulescore/benchmarks',
-    getComponent() {
-      return optionalRequire(() => require('./ModulesBenchmarksScreen'));
-    },
-  });
-}
+// if (!isRunningInExpoGo()) {
```

Comment:
Would be great to figure out why `isRunningInExpoGo` is not working. Maybe `global.expo` is not installed yet?

Reply:
It's working now